### ISSUE TITLE
The checkpoiont is broken

### ISSUE THREAD

**Issue by irfanICMLL** [ericsun99/MobileNet-V2-Pytorch#3]
Hi,
     It is a great work! But the checkpoint is broken. Can you share the model with another way? Or reload the model?

**Comment by miaoshasha** [comment]
yeah, same issue! can't open the tar file! Would you please reload it? thank you!

**Comment by irfanICMLL** [comment]
I have fix the problem. The checkpoint does not need to be extracted. You can load by torch.load('XXX.tar')
